core/app/c/[communitySlug]/pubs/[pubId]/page.tsx

```diff
@@ -140,11 +140,13 @@ export default async function Page({
 									);
 								})}
 							</div>
-							<Move
-								pub={pub}
-								stage={pub.stages[0].stage}
-								communityStages={community.stages}
-							/>
+							{pub.stages[0] ? (
```

Comment:
this is an unrelated bug I found while testing things—a pub could be in no stage, in which case this would try to index into `undefined`!